symbol sets/src/ISOM2017_15000.xmap

```diff
@@ -0,0 +1,10029 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<map xmlns="http://openorienteering.org/apps/mapper/xml/v2" version="6">
+    <notes>ISOM 2017
+Symbols for OpenOrienteering Mapper
+Created by Jukka Liikari and Håvard Tveite, IOF Map Commission
```

Comment:
I created the Open Orienteering Mapper symbol set for ISOM2017 based on an OCAD symbol set that Jukka and several others have contributed to through the years.  I do not claim copyright.  The symbol set was distributed from the IOF to the member federations during the ISOM201X project without any copyright notice.
Please have in mind that the symbol set was created for demonstration purposes. It has not been thoroughly checked and may therefore contain serious mistakes.  It also does not reflect changes that were made before the publishing of the ISOM201X.
Please contact me on email if there are other questions regarding the copyright.

Håvard Tveite